packages/accounts-base/accounts_server.js

```diff
@@ -119,6 +119,22 @@ export class AccountsServer extends AccountsCommon {
 
     this._onCreateUserHook = func;
   }
+
```

Comment:
Can you adjust this so there is only one blank line before and after the `onExternalLogin` function?